account-kit/core/src/metrics.ts

```diff
@@ -0,0 +1,40 @@
+import { createLogger } from "@account-kit/logging";
+import { VERSION } from "./version.js";
+
+export type ConfigCreatedData = {
+  ssr: boolean;
+  chainIds: number[];
+};
+
+export type CoreEventsSchema = [
+  {
+    EventName: "config_created";
```

Comment:
is config created when we have a session id? Should we have an upstream event that is like page load?

Reply:
Yea a session ID (anon ID) in the logger, is available with all event because it get set during initialization of the logger

Reply:
wait... just realize the anonid is per page load. lemme write it to session storage too